addons/sale_timesheet/controllers/portal.py

```diff
@@ -48,8 +48,8 @@ def _get_searchbar_inputs(self):
         searchbar_inputs = super()._get_searchbar_inputs()
         searchbar_inputs.update(
             sol={'input': 'sol', 'label': _('Search in Sales Order Item')},
-            sol_id={'input': 'sol_id', 'label': _('Search in Sales Order Item ID')},
-            invoice={'input': 'invoice_id', 'label': _('Search in Invoice ID')})
+            sol_id={'input': 'so', 'label': _('Search in Sales Order')},
```

Comment:
Shouldn't we name it so instead of sol_id?

Reply:
Yes, we should, my bad ! Thanks !